File: Leetcode/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
       
        
        if(x<0 || (x%10==0 && x!=0)){
            return false;
        }
        
        int sum=0;
        while(x>sum){
            sum = sum*10 + x%10;
            x=x/10;
        }
        return x == sum || x == sum/10;   
        
    }
}

File: LeetCode/palindrome-number.js
let isPalindrome = function(x) {
    var r, temp, final = 0;
    temp = x;
    while (x>0){
        r = x % 10;
        x = parseInt(x / 10);
        final = final * 10 + r;
    }
    if(final == temp){
        return true;
    }else {
        return false;
    }
}